serializations.md

```diff
@@ -15,31 +15,56 @@ user API.
 
 ## Static size object serialization
 
-#### Block header
+There are two types of blocks:
 
-For block headers all fields sizes are statically known and can be
+* key block - does not contain a list of transactions, therefore, it is
+not divided into a header and a list of transactions;
+* micro block - contains a header, a list of transactions and a signature.
+
+### Key block
+
+All field sizes are statically known and can be constructed directly as
+a byte array.
+
+| Fieldname | Size (bytes) |
+| --- | --- |
+| version    | 8    |
+| height     | 8    |
+| prev_hash  | 32   |
+| state_hash | 32   |
+| miner      | 32   |
+| target     | 8    |
+| pow        | 168  |
+| nonce      | 8    |
+| time       | 8    |
```

Comment:
I see in line with https://github.com/aeternity/epoch/pull/1293/files#diff-3422fb1f1624d1021226c01eed4d0cd0R140
Ok.